tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -301,6 +286,94 @@ jobs:
       displayName: Stop Android emulator
       condition: always()
 
+# The below jobs only run on master build
+- job: NNAPI_EP_MASTER
+  pool:
+    vmImage: 'macOS-11'
+  timeoutInMinutes: 120
+  condition: in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')
```

Comment:
You can test branch name instead. Even if a non-master branch posted data to the database, it's ok. We can filter the rows when rendering the data to dashboard.